.travis.yml

```diff
@@ -1,18 +1,29 @@
 ---
 language: python
 dist: xenial
-python:
```

Comment:
Is there a reason why you removed this and instead copied the build stage 3 times? The purpose of this dict was to run everything once for every given Python version, so you basically implemented a Travis Feature yourself ;)

Reply:
I removed the dict at the top, because I don't think, that flake8 has to run with 3 different Python versions, because it only does static code analysis and does not depend on the code.

I tried different variants of using the dictionary to run the build with multiple python versions, but none had functioned. Maybe I could look for a fitting method again, but I am not really optimistic.